test_vm/vagrant_config_defaults.rb

```diff
@@ -2,6 +2,6 @@
   {nfs: true, nfs_version: 4, mount_options: ["noatime", "nodiratime"]}
 
 # Default options
-CONFIG['memory'] = "1100"
+CONFIG['memory'] = "2048"
```

Comment:
I think I've run Oracle+DB2+Informix fine with the 2GB on the guest. I'll see how much I can reduce this before the installers or the tests start failing.